docs/deploy-to-vercel.md

```diff
@@ -1,27 +1,13 @@
 # Deploy to Vercel
 
-[Vercel](https://vercel.com) (formerly ZEIT) is a cloud platform for websites and serverless APIs, that you can use to deploy your Gridsome projects to your personal domain (or a free `.now.sh` suffixed URL).
+[Vercel](https://vercel.com) is a cloud platform that enables developers to host Jamstack websites and web services that deploy instantly, scale automatically, and requires no supervision, all with zero configuration. They provide a global edge network, SSL encryption, asset compression, cache invalidation, and more.
 
-This guide will show you how to get started in a few quick steps:
+### Step 1: Deploying your Gridsome Website to Vercel
 
-### Step 1: Installing Now CLI
+To deploy your Gridsome app with a [Vercel for Git Integration](https://vercel.com/docs/git-integrations), make sure it has been pushed to a Git repository.
 
-To install their command-line interface with [npm](https://www.npmjs.com/package/now), run the following command:
+Import the project into Vercel using the [Import Flow](https://vercel.com/import/git). During the import, you will find all relevant options preconfigured for you; however, you can choose to change any of these options, a list of which can be found [here](https://vercel.com/docs/build-step#build-&-development-settings).
```

Comment:
"you can choose to change" -> "you can change"

Reply:
I updated it to this to make it more clear `During the import, you will find all relevant [options](https://vercel.com/docs/build-step#build-&-development-settings) preconfigured for you.`